lib/tasks/monthly_report.rake

```diff
@@ -21,67 +21,11 @@ end
 
 desc 'Write the latest MonthlyStatisticsReport to JSON and a set of CSV files'
 task export_monthly_report: :environment do
-  stats = Publications::MonthlyStatistics::MonthlyStatisticsReport.last.statistics.to_json
-  File.write('monthly_report.json', stats)
-
-  exports = EXPORTS.map do |export|
```

Comment:
Can we also get rid of the `EXPORTS` array above?

Reply:
yes!